src/__tests__/Rockets.test.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import { render, screen, fireEvent } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import renderer from 'react-test-renderer';
+import store from '../redux/ConfigureStore';
+import Rockets from '../components/Rockets/Rockets';
+
+const ReduxProvider = () => (
+  <Provider store={store}>
+    <Rockets />
+  </Provider>
+);
+
+describe('Check rocket snapshot', () => {
+  it('matches snapshot', () => {
+    const component = renderer
+      .create(
+        <ReduxProvider />,
+      )
+      .toJSON();
```

Comment:
How different is this from the `render()` function of the testing library?

Reply:
Well, using `react-test-renderer` can be based on personal choice, the `renderer. create` allows you to render components and their elements in different forms `toJSON, toTree` etc, but I will do more research on their similarities and differences.